src/components/BaseSeparatePage/index.tsx

```diff
@@ -84,24 +83,25 @@ export const BaseSeparatePage: FC<BaseSeparatePageType> = props => {
   }) => (
     <div className={styles.functionsWrap}>
       <Functions isProgressBar={isProgressBar} functions={functions} className={className} />
-      {/* Todo: progressbar need complete later*/}
-      {isProgressBar ? <TableOfContents className={clsx(styles.tableOfContents)} editLink={editLink} /> : null}
+      {/* {isProgressBar ? <TableOfContents className={clsx(styles.tableOfContents)} editLink={editLink} /> : null} */}
```

Comment:
Is it written wrong here?

Reply:
in our original logic, the `FunctionsContainer ` need to be compatible with and without a TOC.

Reply:
I mean it doesn't look like it should be annotated out here?
If it is annotated, there will be no opportunity to show the TOC.